presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -269,6 +272,9 @@ public BaseJdbcClient(
                             .setComment(comment)
                             .build());
                 }
+                else {
+                    verify(getUnsupportedTypeHandling(session) == IGNORE, "UnsupportedTypeHandling should be handled in toPrestoType method");
```

Comment:
`"Unsupported type handling is set to %s, but toPrestoType() returned empty"`

also 

`verify(columnMapping.isPresent() || getUnsupportedTypeHandling(session) == IGNORE, ..` so that you can place it above, without `else`